tests/Unit/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Test_Tov.cpp

```diff
@@ -108,62 +108,6 @@ void test_tov(
         custom_approx(interpolated_log_enthalpy_ds));
 }
 
-void test_tov_dp_dr(
```

Comment:
Why remove this test? This was added because it actually caught a bug and was really useful for debugging GRMHD with FD.

Reply:
I'll add it back 👍 Can't remember why I removed it